configs/components/libedit.rb

```diff
@@ -11,9 +11,8 @@
   elsif platform.is_aix?
     pkg.environment "CC", "/opt/pl-build-tools/bin/gcc"
     pkg.environment "LDFLAGS", settings[:ldflags]
-  end
-
-  if platform.is_macos?
+  else
+    pkg.environment "LDFLAGS", settings[:ldflags]
```

Comment:
If we build libedit for more platforms than solaris, aix and mac, then this will result in a behavior change for those other platforms. Though possibly that is intended so that the CFLAGS and LDFLAGS environment variables are set?

Reply:
nevermine